### ISSUE TITLE
Browser Sync / Reload Options

### ISSUE THREAD

**Issue by bgrrtt** [everysquare/craft-ops#6]
Unless there is something built-in that I am unaware of, which is totally possible, the only piece of the dev setup that seems to be missing is browser-sync/auto-reload functionality. Maybe you were leaving that out so Craft-Ops can be agnostic of that. Curious about your favorite solution though.

Currently using browser-sync with a proxy of localhost:8000.

Is there something better? THANKS!!

**Comment by howardroark** [comment]
To be honest I have never thought to try that kind of thing out.  Go ahead and add something :)

If you take a look in `salt/root/dev/init.sls` you will see the `supervisor` task list. Install any module you like, and start it up there. Preferably it installs globally and has some sort of command line interface.

browser-sync sounds great! I'll try it out. Submit a pull request with a little info about how to use it in context with Craft.  That would be nice ;)

**Comment by howardroark** [comment]
So I started to play with `browser-sync`.  Out of the box there is no real support for it through Harp.  Though if we ran Harp with Gulp it could be rigged to watch a CSS files for changes to enable streaming updates.  It seems like the real win with browser-sync is that ability.

The the idea would be to proxy the port 8000 nginx process through browser-sync on the VM and port forward 3000 to the host machine.  This would give you a port 3000 version of the site that streams in CSS changes.
